### DISCUSSION TITLE
Feature Request: SysVar-Entitäten und WebUI-Programm-Entitäten separat aktivieren/deaktivieren

### DISCUSSION THREAD

**Discussion by Baxxy13** [SukramJ/aiohomematic#1717] [Q&A] [answered]
Als "follow up" zu #1716 ist mir aufgefallen das das aktivieren/deaktivieren des SysVar-Scans gleichzeitig das Erzeugen der WebUI-Programm-Entitäten beeinflusst.

Starte ich mit 1.64.0 und deaktiviere im "ersten Durchlauf" durch den Config-Flow den SysVar-Scan dann werden weder Entitäten für die SysVars noch für die WebUI-Programme angelegt. 
Aktiviere ich hingegen den SysVar-Scan so werden die SysVar-Entitäten und die WebUI-Programm-Entitäten erzeugt.

Ich persönlich brauche weder die SysVars in HA noch die WebUI-Programme, mir ist aber klar das manch anderer das Eine oder Andere (oder beides) gerne hätte.

Ich denke es wäre nützlich wenn man beides separat aktivieren/deaktivieren könnte.

**Comment by SukramJ** [comment, marked as answer]
Gute Idee. Hab ich mit der [1.66.0b8](https://github.com/danielperna84/custom_homematic/releases/tag/1.66.0b8) umgesetzt
